src/main/kotlin/com/aemtools/completion/htl/completionprovider/FileVariablesResolver.kt

```diff
@@ -127,11 +128,9 @@ object FileVariablesResolver {
                     startsWith(DATA_SLY_LIST) -> {
                         val (itemName, itemListName) = extractItemAndItemListNames(this)
 
-                        val itemClass = resolveClassForDataSlyList(it)
-
                         val tag = it.findParentByType(XmlTag::class.java) ?: return result
 
-                        if (position.within(tag)) {
+                        if (position.isWithin(tag)) {
```

Comment:
I think we need rule that ALL public methods should have documentation and, in case with htl, even with examples like you do in some methods.